src/extension/mapml/src/main/java/org/geoserver/mapml/MapMLURLBuilder.java

```diff
@@ -613,56 +616,27 @@ private String getSupportedWMTSGridSet(
             }
 
             // check same scales
-            for (int i = 0; i < tileMatrices.size(); i++) {
-                if (Math.abs(tileMatrices.get(i).getDenominator() - tiledCRSScales[i])
+            boolean sameScale = true;
+            for (int i = 0; i < tiledCRSResolutions.length; i++) {
+                if (Math.abs(tileMatrices.get(i).getResolution() - tiledCRSResolutions[i])
```

Comment:
Everything was renamed to "resolution" (is it actually) but it's still compared to a SCALE_DELTA. So it is a pixel resolution, a scale denominator, a scale... what?